reframe/core/logging.py

```diff
@@ -385,7 +417,9 @@ def debug2(self, message, *args, **kwargs):
 
 class LoggerAdapter(logging.LoggerAdapter):
     def __init__(self, logger=None, check=None):
-        extra_vars = {
+        super().__init__(
+            logger,
+            {
                 'check_name': 'reframe',
```

Comment:
You still need to remove from here all the extras that correspond to an actual test attribute.